std/meta.d

```diff
@@ -61,6 +61,67 @@ unittest
 }
 
 /**
+ * Not everything can be directly aliased. An alias cannot be declared
```

Comment:
I think the first paragraph of documentation should always say, in preferably a single sentence, what the type/function/template is or does. Here you have to scan several lines down to see what `Alias` does.